templates/frontend/components/objectFiles.tpl

```diff
@@ -0,0 +1,74 @@
+{**
+ * templates/frontend/components/objectFiles.tpl
```

Comment:
In line `{include file="frontend/components/publicationFormats.tpl" publicationFormats=$bookFiles}` I've too change `publicationFormats=$bookFiles` to `publicationFiles=$bookFiles`, because `publicationFormats` is already used in this context.